src/rebar_prv_common_test.erl

```diff
@@ -256,8 +256,13 @@ do_compile(State) ->
     case rebar_prv_compile:do(State) of
         %% successfully compiled apps
         {ok, S} ->
-            ok = maybe_cover_compile(S),
-            {ok, S};
+            case rebar_prv_app_builder:do(S) of
```

Comment:
That is what the enclosing case statement does. eunit and bare_compile are covered in later commits.